pkg/converter/linker_test.go

```diff
@@ -9,76 +9,70 @@ import (
 	"github.com/sloppyio/sloppose/pkg/converter"
 )
 
-var cfA *converter.ComposeFile
-var sfA *converter.SloppyFile
+func TestLinker_FindService(t *testing.T) {
+	cases := map[string]struct {
+		value       string
+		shouldMatch bool
+	}{
+		"FOO":      {"bar", false},
+		"BAR":      {"foo:80", true},
+		"SHORT":    {"s:80", true},
+		"FOO_BAR":  {"another.foo:443", true},
+		"FOO_HOST": {"bar", true},
```

Comment:
Requiring the string 'HOST' in the env var key is a pretty harsh limitation to. Took me a while to understand that based on the implementation. Need to at least add this to the readme, maybe a "Limitations" section.